src/app/components/editor/autocomplete/EmoticonAutocomplete.tsx

```diff
@@ -57,10 +57,12 @@ export function EmoticonAutocomplete({
 
   const searchList = useMemo(() => {
     const list: Array<EmoticonSearchItem> = [];
-    return list.concat(
-      imagePacks.flatMap((pack) => pack.getImagesFor(PackUsage.Emoticon)),
-      emojis
-    );
+    return list
+      .concat(
+        imagePacks.flatMap((pack) => pack.getImagesFor(PackUsage.Emoticon)),
+        emojis
+      )
+      .sort((a, b) => a.shortcode.localeCompare(b.shortcode));
   }, [imagePacks]);
 
   const [result, search, resetSearch] = useAsyncSearch(searchList, getEmoticonStr, SEARCH_OPTIONS);
```

Comment:
From the performance point of view, I think instead of sorting the `searchList` we should sort the `result.item`.

Reply:
Makes sense, I'll update this in about 2 days when I get to my computer.

Reply:
Sorry for the delay, done as requested